todo_app/app.py

```diff
@@ -1,38 +1,58 @@
 from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 from todo_app.data.trello_items import Trello
-
-app = Flask(__name__)
-app.config.from_object(Config())
-trello = Trello()
-
-@app.route('/')
-def index():
-    items = trello.get_items()
-    return render_template('index.html', items=items)
-
-
-@app.route('/items/new', methods=['POST'])
-def new_item():
-    title = request.form.get('item_title')
-    status = request.form.get('status')
-    trello.add_item(status, title)
-    return redirect(url_for('index'))
-
-
-@app.route('/<item_id>/done', methods=['POST'])
-def move_to_done(item_id):
-    trello.change_status(item_id, status='Done')
-    return redirect(url_for('index'))
-
-
-@app.route('/<item_id>/in_progress', methods=['POST'])
-def move_to_in_progress(item_id):
-    trello.change_status(item_id, status='In Progress')
-    return redirect(url_for('index'))
-
-
-@app.route('/<item_id>/to_do', methods=['POST'])
-def move_to_to_do(item_id):
-    trello.change_status(item_id, status='To Do')
-    return redirect(url_for('index'))
+from todo_app.view_model import ViewModel
+
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+    trello = Trello()
+
+    @app.route('/')
+    def index():
+        items = trello.get_items()
+        item_view_model = ViewModel(items)
+        return render_template('index.html', view_model=item_view_model.items)
```

Comment:
Really minor, but if you're going to pass through just the items themselves, rather than the ViewModel wrapper, then I'd suggest naming it as such to avoid confusion, i.e. something like:
```python
        return render_template('index.html', items=item_view_model.items)
```

Reply:
Updated